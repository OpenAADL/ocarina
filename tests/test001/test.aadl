-- test for parsing package

package toto::titi::tata
   public
   	process tutu
   	end tutu;
   private
   	process tete
   	end tete;
end toto::titi::tata;

package toto
end toto;

package toto::titi
end toto::titi;