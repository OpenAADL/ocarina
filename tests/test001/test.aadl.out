test.aadl:13:01: parsing Package_Specification, token 'public' or 'private' is expected, found keyword 'end'
test.aadl:16:01: parsing Package_Specification, token 'public' or 'private' is expected, found keyword 'end'
Cannot parse AADL specifications
